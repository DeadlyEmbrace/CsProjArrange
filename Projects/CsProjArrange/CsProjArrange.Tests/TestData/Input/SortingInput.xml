<Project ToolsVersion="12.0" DefaultTargets="Build">
  <ItemGroup>
    <Compile Include="Program.cs" />
    <Compile Include="Properties\AssemblyInfo.cs" />
    <Compile Include="CsProjArrange.cs" />
    <Compile Include="Mono.Options\Options.cs" />
    <Compile Include="CsProjArrangeConsole.cs" />    
  </ItemGroup>
  <ItemGroup>
    <Compile Include="A.cs" />
    <Compile Include="Z.cs" />
    <Compile Include="D.cs">
      <SubElement Include="E.cs"/>
      <SubElement Include="A.cs"/>
    </Compile>
    <Compile Include="F.cs" />
    <Compile Include="B.cs" />
  </ItemGroup>
</Project>