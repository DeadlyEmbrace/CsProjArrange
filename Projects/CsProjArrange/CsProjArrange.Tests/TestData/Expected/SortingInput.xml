<Project ToolsVersion="12.0" DefaultTargets="Build">
  <ItemGroup>
    <Compile Include="CsProjArrange.cs" />
    <Compile Include="CsProjArrangeConsole.cs" />
    <Compile Include="Mono.Options\Options.cs" />
    <Compile Include="Program.cs" />
    <Compile Include="Properties\AssemblyInfo.cs" />
  </ItemGroup>
  <ItemGroup>
    <Compile Include="A.cs" />
    <Compile Include="B.cs" />
    <Compile Include="D.cs">
      <SubElement Include="A.cs"/>
      <SubElement Include="E.cs"/>
    </Compile>
    <Compile Include="F.cs" />
    <Compile Include="Z.cs" />
  </ItemGroup>
</Project>